src/interfaces/ecpg/test/expected/meson.build

```diff
@@ -0,0 +1,207 @@
+  'preproc-pointer_to_struct.c',
+  'preproc-pointer_to_struct.stderr',
+  'preproc-pointer_to_struct.stdout',
+  'preproc-strings.c',
+  'preproc-strings.stderr',
+  'preproc-strings.stdout',
+  'preproc-type.c',
+  'preproc-type.stderr',
+  'preproc-type.stdout',
+  'preproc-variable.c',
+  'preproc-variable.stderr',
+  'preproc-variable.stdout',
+  'preproc-whenever.c',
+  'preproc-whenever.stderr',
+  'preproc-whenever.stdout',
+  'preproc-whenever_do_continue.c',
+  'preproc-whenever_do_continue.stderr',
+  'preproc-whenever_do_continue.stdout',
+  'sql-array.c',
+  'sql-array.stderr',
+  'sql-array.stdout',
+  'sql-binary.c',
+  'sql-binary.stderr',
+  'sql-binary.stdout',
+  'sql-bytea.c',
+  'sql-bytea.stderr',
+  'sql-bytea.stdout',
+  'sql-code100.c',
+  'sql-code100.stderr',
+  'sql-code100.stdout',
+  'sql-copystdout.c',
+  'sql-copystdout.stderr',
+  'sql-copystdout.stdout',
+  'sql-createtableas.c',
+  'sql-createtableas.stderr',
+  'sql-createtableas.stdout',
+  'sql-declare.c',
+  'sql-declare.stderr',
+  'sql-declare.stdout',
+  'sql-define.c',
+  'sql-define.stderr',
+  'sql-define.stdout',
+  'sql-desc.c',
+  'sql-desc.stderr',
+  'sql-desc.stdout',
+  'sql-describe.c',
+  'sql-describe.stderr',
+  'sql-describe.stdout',
+  'sql-dynalloc.c',
+  'sql-dynalloc.stderr',
+  'sql-dynalloc.stdout',
+  'sql-dynalloc2.c',
+  'sql-dynalloc2.stderr',
+  'sql-dynalloc2.stdout',
+  'sql-dyntest.c',
+  'sql-dyntest.stderr',
+  'sql-dyntest.stdout',
+  'sql-execute.c',
+  'sql-execute.stderr',
+  'sql-execute.stdout',
+  'sql-fetch.c',
+  'sql-fetch.stderr',
+  'sql-fetch.stdout',
+  'sql-func.c',
+  'sql-func.stderr',
+  'sql-func.stdout',
+  'sql-indicators.c',
+  'sql-indicators.stderr',
+  'sql-indicators.stdout',
+  'sql-insupd.c',
+  'sql-insupd.stderr',
+  'sql-insupd.stdout',
+  'sql-oldexec.c',
+  'sql-oldexec.stderr',
+  'sql-oldexec.stdout',
+  'sql-parser.c',
+  'sql-parser.stderr',
+  'sql-parser.stdout',
+  'sql-prepareas.c',
+  'sql-prepareas.stderr',
+  'sql-prepareas.stdout',
+  'sql-quote.c',
+  'sql-quote.stderr',
+  'sql-quote.stdout',
+  'sql-show.c',
+  'sql-show.stderr',
+  'sql-show.stdout',
+  'sql-sqlda.c',
+  'sql-sqlda.stderr',
+  'sql-sqlda.stdout',
+  'sql-twophase.c',
+  'sql-twophase.stderr',
+  'sql-twophase.stdout',
+  'thread-alloc.c',
+  'thread-alloc.stderr',
+  'thread-alloc.stdout',
+  'thread-alloc_2.stdout',
+  'thread-descriptor.c',
+  'thread-descriptor.stderr',
+  'thread-descriptor.stdout',
+  'thread-prep.c',
+  'thread-prep.stderr',
+  'thread-prep.stdout',
+  'thread-prep_2.stdout',
+  'thread-thread.c',
+  'thread-thread.stderr',
+  'thread-thread.stdout',
+  'thread-thread_2.stdout',
+  'thread-thread_implicit.c',
+  'thread-thread_implicit.stderr',
+  'thread-thread_implicit.stdout',
+  'thread-thread_implicit_2.stdout',
+]
+
+foreach copy_file: copy_files_list
+  configure_file(
+    output: '@0@'.format(copy_file),
+    input: '@0@'.format(copy_file),
+    copy : true
+  )
+endforeach
```

Comment:
I think it'd be strongly preferrable to avoid the need for this copying by adjusting the test infrastructure to find the file in their original place. As far as I can see ecpg's pg_regress already has --inputdir,  right?

Reply:
+1

Depending on the project I've sometimes also been able to get it to find files by setting the `workdir` kwarg to run in the source directory instead of the build root

Reply:
The problem is, when we set --inputdir; pg_regress looks for expected files and .c, exe files in the --inputdir directory. Expected files are in the source directory but .c and exe files are in the build directory(since these are generated by using executable and custom_target). 

Maybe we can add something like --expecteddir to the pg_regress?

Reply:
> The problem is, when we set --inputdir; pg_regress looks for expected files and .c, exe files in the --inputdir directory. Expected files are in the source directory but .c and exe files are in the build directory(since these are generated by using executable and custom_target).

Ah, I see.

> Maybe we can add something like --expecteddir to the pg_regress?

That sounds like a good plan.

Reply:
Updated.